A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -784,47 +796,64 @@ class CDUPerformancePage {
                 CDUPerformancePage.ShowAPPRPage(mcdu);
             }
         };
-        mcdu.onRightInput[3] = () => {
-            mcdu.setPerfApprFlaps3(true);
-            mcdu.updatePerfSpeeds();
-            CDUPerformancePage.ShowAPPRPage(mcdu);
-        };
         mcdu.onRightInput[4] = () => {
-            mcdu.setPerfApprFlaps3(false);
+            mcdu.setPerfApprFlaps3(!mcdu.perfApprFlaps3);
             mcdu.updatePerfSpeeds();
             CDUPerformancePage.ShowAPPRPage(mcdu);
         };
-        let finalCell = "-----";
+
+        let finalCell = "";
         const approach = mcdu.flightPlanManager.getApproach();
         if (approach && approach.name) {
-            finalCell = Avionics.Utils.formatRunway(approach.name);
+            finalCell = `\xa0{green}${Avionics.Utils.formatRunway(approach.name).replace(/ /g, "")}{end}\xa0\xa0\xa0\xa0\xa0`;
         }
-        let mdaCell = "[ ]";
+
+        let baroCell = "[\xa0\xa0\xa0]";
         if (isFinite(mcdu.perfApprMDA)) {
-            mdaCell = mcdu.perfApprMDA.toFixed(0);
+            baroCell = mcdu.perfApprMDA.toFixed(0);
         }
         mcdu.onRightInput[1] = (value) => {
             if (mcdu.setPerfApprMDA(value) && mcdu.setPerfApprDH(FMCMainDisplay.clrValue)) {
```

Comment:
As mentioned already, this needs to be limited to (airfield elevation to airfield elevation + 5000ft) and (0 to 5000) respectively.

Reply:
iircf clamp(0, 5000, value)